web/spiders/spider_ce.py

```diff
@@ -0,0 +1,65 @@
+import io
+from datetime import datetime
+
+import rows
+
+from .base import BaseCovid19Spider
+
+
+class Covid19CESpider(BaseCovid19Spider):
+    name = "CE"
+    start_urls = ["https://indicadores.integrasus.saude.ce.gov.br/api/coronavirus/qtd-por-municipio?tipo=Óbito,Confirmado"]
+
+    def parse(self, response):
+        encoding = "utf-8"
+        table = rows.import_from_json(
+            io.BytesIO(response.body),
+            encoding=encoding
+        )
+
+        # TODO - get last_date from another source at saude.ce.gov.br
+        last_date = datetime.now().date()
+        self.add_report(date=last_date, url=self.start_urls[0])
+
+        total_confirmed = total_deaths = 0
+        imported_confirmed = imported_deaths = None
+
+        cases = {}
+        for row in table:
+
+            if cases.get(row.municipio) == None :
+                cases[row.municipio] = {}
+
+            if row.tipo == 'Confirmado':
+                cases[row.municipio]["confirmed"] = row.quantidade
+                continue
+            
+            if row.tipo == 'Óbito':
+                cases[row.municipio]["deaths"] = row.quantidade 
+                continue
+        
+        for case in cases.items():
+            city = case[0]
+            confirmed = case[1]["confirmed"]
+            deaths = case[1].get("deaths", 0)
+
+            try:
+                self.get_city_id_from_name(city)
+            except KeyError:
```

Comment:
Aqui, novamente uma sugestão de código defensivo: se esse `KeyError` acontecer mais de uma vez, teremos as variáveis `imported_confirmed` e `imported_deaths` sobrescritas a partir da segunda vez. Dessa forma, o ideal seria iniciá-las com o valor `0` e nessa Exception usar `+=` em vez de `=` (se elas continuarem com `0` depois do `for`, acho que poderíamos assumir que os dados não foram publicados e daí transformar, com um `if`, esses `0`s em `None`, que representa ausência de dados).

Reply:
inicializei com `0` e adicionei `+=`

não entendi onde fazer esse ajuste do `if` e `None` q vc comentou no final